src/main/java/io/kamax/matrix/json/event/MatrixJsonRoomTagsEvent.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * matrix-java-sdk - Matrix Client SDK for Java
+ * Copyright (C) 2018 Arne Augenstein
+ *
+ * https://www.kamax.io/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.kamax.matrix.json.event;
+
+import com.google.gson.JsonObject;
+
+import io.kamax.matrix.event._RoomTagsEvent;
+import io.kamax.matrix.json.GsonUtil;
+import io.kamax.matrix.json.MatrixJsonObject;
+import io.kamax.matrix.room.RoomTag;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MatrixJsonRoomTagsEvent extends MatrixJsonObject implements _RoomTagsEvent {
+
+    private Content content;
+
+    private String type;
+
+    public MatrixJsonRoomTagsEvent(JsonObject obj) {
+        super(obj);
+        type = getString("type");
+        content = new Content(getObj("content"));
+    }
+
+    @Override
+    public String getType() {
+        return type;
+    }
+
+    private class Content extends MatrixJsonObject {
+
+        private List<RoomTag> tags = new ArrayList<>();
+
+        public Content(JsonObject obj) {
+            super(obj);
+
+            String roomId = getStringOrNull("roomId");
+
+            GsonUtil.findObj(obj, "tags").ifPresent(tagsJson -> {
+                List<RoomTag> tags = tagsJson.entrySet().stream().map(it -> {
+                    String completeName = it.getKey();
+                    String name = completeName;
+                    String namespace = "";
+                    if (completeName.charAt(1) == '.') {
```

Comment:
That's only true for `u` and `m` namespace, but not necessarly for [the arbitrary namespaces](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#id75):

> A client or app willing to use special tags for advanced functionnality should namespace them similarly to state keys: tld.name.*

While dumb, I can see a tld of one letter being used anyway...